data/tools/unit_tree/html_output.py

```diff
@@ -477,8 +481,8 @@ def grid_place(nodes, x):
     def write_navbar(self, report_type):
         def write(line):
             self.output.write(line)
-        def _(msgid, textdomain="wesnoth"):
-            return self.translate(msgid, textdomain)
+
+        _w, _w_editor, _w_help, _w_lib = self.make_gettext("wesnoth", "wesnoth-editor", "wesnoth-help", "wesnoth-lib")
```

Comment:
Putting them on a single line is less readable, IMO. Also, it means the entire line changes if a domain is added or removed (as I hope _w_editor will be in a subsequent PR).

Reply:
Is this really superior?

```python
_w = self.make_gettext("wesnoth")
_w_editor = self.make_gettext("wesnoth-editor")
_w_help = self.make_gettext("wesnoth-help")
_w_lib = self.make_gettext("wesnoth-lib)
```
Remember that snippet would have to be included more than once (in fact, 3 times -the current state of the PR only 2.)

Reply:
If the worry is about changing the entire line just for one domain, then what about this instead?

```python
_w,
_w_editor,
_w_help,
_w_lib =
    self.make_gettext(
        "wesnoth",
        "wesnoth-editor",
        "wesnoth-help",
        "wesnoth-lib"
    )
```

Not the most pretty formatting, but does avoid touching the whole statement just to add or remove a domain.

I'm not sure if a `\` is also needed at the end of the first 3 lines – I know `=` and `(` trigger statement continuation but I'm not sure if `,` does.

Reply:
I just did a quick test and `\` was needed for the first four lines - including the `=`.